staging/cert-manager-setup/templates/v1upgrade.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: cert-manager-upgrade-backup
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-6"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    spec:
+      serviceAccountName: cert-manager-upgrade
+      containers:
+        - name: cert-manager-upgrade
+          image: "jaredrodriguez/cert-manager-upgrade:v0.1.0"
```

Comment:
are we moving these to prod repo?

Reply:
The plan is yes. We were having a bit of a discussion where to - considered extra-steps but I think many want to deprecate it so we may create a new repo instead (perhaps you have the power to create new repos? 💪 ).

Reply:
i might know a guy 😏

Reply:
If we need something fast, extra-steps seems like a legit place, just saying.